package/MDAnalysis/coordinates/__init__.py

```diff
@@ -720,13 +720,3 @@ class can choose an appropriate reader automatically.
 from . import memory
 from . import MMTF
 from . import null
-
```

Comment:
If you add these imports back in, more tests should start passing (or failing with better messages)